lib/recart-nacelle-plugin.js

```diff
@@ -1,5 +1,5 @@
 const { getMetafieldsForCheckout, saveCheckoutId } = require('./checkout')
-const { saveCart } = require('./cart')
+const cartLib = require('./cart')
```

Comment:
```suggestion
const { rebuildCart, saveCart } = require('./cart')
```

Reply:
Had to change to be able to stub for tests. Cant use destructuring on require.